src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @flow
+ * @file Versions Sidebar component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+
+import { BackButton } from '../../common/nav-button';
+import Button from '../../../components/button';
+import BoxDrive140 from '../../../illustration/BoxDrive140';
+
+import { LoadingIndicatorWrapper } from '../../../components/loading-indicator';
+
+import VersionsMenu from './VersionsMenu';
+import './StaticVersionsSidebar.scss';
+
+type Props = {
+    isLoading: boolean,
+    onUpgradeClick?: () => void,
+    parentName: string,
+    showUpsellWithPicture: boolean,
+};
+
+const StaticVersionsSidebar = ({ onUpgradeClick, isLoading, parentName, showUpsellWithPicture }: Props): Object => {
+    const yesterday = new Date();
+    yesterday.setDate(yesterday.getDate() - 1);
+    const versionTimestamp = new Date(yesterday);
```

Comment:
const yesterday was set to the date for yesterday, this line might not be needed?